test/compose.sh

```diff
@@ -277,4 +277,4 @@ test-quorum-consistency() {
 test-rejoin-raft 3
 test-rejoin-raft 5
 test-graceful-leave
-test-quorum-consistency
+#test-quorum-consistency
```

Comment:
If you're going to leave this in, but commented out, can you add to the comment and explain why.

Reply:
You're right, I haven't seen it work even on the previous versions but that probably deserves more of an explanation.

Reply:
Spent more than 2 minutes looking at the problem and it appears the output changed between versions. Patch incoming. 👍